lib/Dialect/Secret/Transforms/DistributeGeneric.cpp

```diff
@@ -543,6 +543,8 @@ struct DistributeGeneric
       }
     });
 
+    opsToDistribute.push_back("memref.get_global");  // TEMP FIXME
```

Comment:
Resolve, remove, or convert FIXME comment to TODO

Reply:
Yes! This is still a draft PR, I just meant to push the intermediate pipeline that I'm getting my testcases from. This issue in particular is: https://github.com/google/heir/issues/352